app/controllers/application_controller.rb

```diff
@@ -89,6 +90,13 @@ def recognized_member
                            Member.find_from_request(akid: unsafe_params[:akid], id: cookies.signed[:member_id])
   end
 
+  def id_mismatch
+    return false if recognized_member.nil?
+
+    Rails.logger.info("current is '#{current_member}' and recognized is '#{@recognized_member.id}'")
```

Comment:
I added this for debugging/testing purposes and I will remove it in a follow-up PR after I test this on staging.